examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+# subject's session from the HARDI data into fetcher.dipy_home:
+# `~/.dipy/stanford_hardi`
+dpd.fetch_stanford_hardi()
+
+# Reference to data file locations
+hardi_dir = op.join(fetcher.dipy_home, "stanford_hardi")
+hardi_fdata = op.join(hardi_dir, "HARDI150.nii.gz")
+hardi_fbval = op.join(hardi_dir, "HARDI150.bval")
+hardi_fbvec = op.join(hardi_dir, "HARDI150.bvec")
+
+print(f'Loading data file: {hardi_fdata}')
+img = nib.load(hardi_fdata)
+
+# Display output from this step
+img_data = img.get_fdata()
+
+# working with space and time data
+img_data = img_data[..., int(img_data.shape[3] / 2)]
+
+show_anatomical_slices(img_data, 'HARDI 150 DWI')
+print(f'bvals: f{np.loadtxt(hardi_fbval)}')
+print(f'bvec: f{np.loadtxt(hardi_fbvec)}')
+
+
+##########################################################################
+# Calculate DTI:
+# --------------
+# Fit the DTI model using default settings, save files with derived maps.
+#
+# The by default following DTI measurements are calculated:
+#
+# - Fractional anisotropy (FA),
+#
+# - Mean diffusivity (MD),
+#
+# - Axial diffusivity (AD),
+#
+# - and Radial diffusivity (RD)
+#
+# In this example we will only use FA.
+#
+# .. note::
+#   By default:
+#
+#   - All b-values less than or equal to 50 are considered to be
+#     without diffusion weighting.
+#
+#   - No binary masks are applied; therefore all voxels are processed.
+#
+# .. note::
+#   The diffusion tensor imaging parameters contain the associated eigenvalues
+#   and eigenvectors from eigen decomposition on the diffusion tensor.
+
+
+print("Calculating DTI...")
+
+if not op.exists(op.join(working_dir, 'dti_FA.nii.gz')):
+    dti_params = dti.fit_dti(hardi_fdata, hardi_fbval, hardi_fbvec,
+                             out_dir=working_dir)
+else:
+    dti_params = {'FA': op.join(working_dir, 'dti_FA.nii.gz'),
+                  'params': op.join(working_dir, 'dti_params.nii.gz')}
+
+print(f"Loading {dti_params['FA']}")
+FA_img = nib.load(dti_params['FA'])
+FA_data = FA_img.get_fdata()
+
+show_anatomical_slices(FA_data, 'Fractional Anisotropy (FA)')
+
+##########################################################################
+# Register the individual data to a template:
+# -------------------------------------------
+# For the purpose of bundle segmentation, the individual brain is registered
+# to the MNI T2 template. The waypoint ROIs used in segmentation are then each
+# brought into each subject's native space to test streamlines for whether they
+# fulfill the segmentation criteria.
+#
+# .. note::
+#    Here, for simplicity we only perform a non-linear transformation between
+#    the individual's brain and the MNI T2 template. This is likely why the
+#    occipital/parietal junction appears concave. In practice, it's a good idea
+#    to also perform a pre-alignment using an affine transformation. Moreover,
+#    by default, registration uses APM (Anisotropic Power Map) which also tends
+#    to provide better results.
```

Comment:
Add a registration example to demonstrate effect of different registrations on process

1. using alternative (non-MNI T2) registration template
2. including/excluding pre-alignment and affine transformation
3. using APM for registration (see models/csd.py fit_anisotropic_power_map())

Reply:
@jyeatman I may be misstating this, but believe you mentioned testing out using APM in bundle classification instead of FA. If this is accurate I was wondering if could elaborate.

Reply:
For items 1 and potentially 2 see #487